### PR TITLE
Remove CapabilityTag

### PR DESCRIPTION
**Pull request by clemens-msupply**
Part of #1944

# 👩🏻‍💻 What does this PR do? 
Removes the CapabilityTag enum. Contexts are unique now which makes the (theortical) problem that CapbilityTag solved go away...

### CODE REVIEW

File: server/context_table.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Context Table
```
Comment (clemens-msupply) [inline server/context_table.md:1]: Better place for this file?

Comment (andreievg) [reply server/context_table.md:1]: Sounds like a good place for now, thanks, some [suggested fixes](https://github.com/openmsupply/open-msupply/blob/2a3d571f4a6d1086e3356098595da5b8e8f08563/server/context_table.md)

File: server/service/src/auth.rs [hunk 2]
```diff
@@ -12,20 +12,11 @@ use crate::{
...
-#[derive(Debug, Clone, Eq, Hash, PartialEq)]
-pub enum CapabilityTag {
-    /// Tags the list of capabilities to access documents by type
-    ContextType,
-}
-
 #[derive(Debug, Clone)]
 pub enum PermissionDSL {
     HasPermission(Permission),
     /// The permission context will be extracted and tagged with the provided tag.
```
Comment (andreievg) [inline server/service/src/auth.rs:18]: Is this comment still valid ?

**Review by andreievg — COMMENTED** [review]
Hi looks good, would approve but have a question about this line: 

> Currently, the permission for this Patient context is automatically added (in memory) when the user has the correct mSupply patient editing permission.

I couldn't see where this happens (is there another PR for this) ?

To answer the above I did a search for patient_context_id and 'Patient'. I found two types "PATIENT_TYPE" and "PATIENT_CONTEXT_ID" do you think it's a good idea to consolidate those ? maybe in 
[constants.rs](https://github.com/openmsupply/open-msupply/blob/6205bc0361cfc144e9969d97b17c0bb71b08caf5/server/util/src/constants.rs)

File: server/service/src/auth.rs (hunk 2 above)
Comment (clemens-msupply) [reply server/service/src/auth.rs:18]: thanks I will update in https://github.com/openmsupply/open-msupply/pull/2047

**Comment by clemens-msupply** [comment]
> Hi looks good, would approve but have a question about this line:
> 
> > Currently, the permission for this Patient context is automatically added (in memory) when the user has the correct mSupply patient editing permission.
> 
> I couldn't see where this happens (is there another PR for this) ?
> 
> To answer the above I did a search for patient_context_id and 'Patient'. I found two types "PATIENT_TYPE" and "PATIENT_CONTEXT_ID" do you think it's a good idea to consolidate those ? maybe in [constants.rs](https://github.com/openmsupply/open-msupply/blob/6205bc0361cfc144e9969d97b17c0bb71b08caf5/server/util/src/constants.rs)

I moved the constants and actually use the constants in auth.rs. Have a look at auth.rs in [f802ae2](https://github.com/openmsupply/open-msupply/pull/2047/commits/f802ae2f5c39cf24d11d7fc2abcca646b16cc77b) for the added permisisons.

**Comment by andreievg** [comment]
As per telegram chat:

The following branch had all of the context changes: https://github.com/openmsupply/open-msupply/compare/develop...1944-review-comments (+1.6k - 1.2k).
But it was merged here: https://github.com/openmsupply/open-msupply/pull/2047, to base branch but there are soo many changes

I think it's best to just merge 1944-review-comments to develop (I can approve it pretty quickly). I was going to approve the next PR to be merged, but not sure why the changes are so big: https://github.com/openmsupply/open-msupply/pull/2017

**Pull request merged**